zipline.gemspec

```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |gem|
   gem.version       = Zipline::VERSION
   gem.licenses      = ['MIT']
 
-  gem.add_dependency 'zip_tricks', ['>= 4.2.1', '<= 5.1.0']
+  gem.add_dependency 'zip_tricks', ['>= 4.2.1', '<= 6.0']
```

Comment:
this must be '< 6.0' or '<= 5.3', presumable 6.0 would include breaking changes.

Reply:
Ah yes, good catch. I'll change it.